docs/5.0/preview/database-access.md

```diff
@@ -38,6 +38,134 @@ With Database Access users can:
 Your browser does not support the video tag.
 </video>
 
+## Getting Started
+
+In this guide we will use Teleport Database Access to connect to a PostgreSQL
+flavored AWS Aurora database. Here's an overview of what we will do:
+
+1. Configure AWS Aurora database with IAM authentication.
+2. Download and install Teleport and connect it to the Aurora database.
+3. Connect to the Aurora database via Teleport.
+
+### Step 1/3. Setup Aurora
+
+In order to allow Teleport connections to an Aurora instance, it needs to support
+IAM authentication.
+
+If you don't have a database provisioned yet, create an instance of an Aurora
+PostgreSQL instance in the [RDS control panel](https://console.aws.amazon.com/rds/home).
+Make sure to enable "Password and IAM database authentication" in the Database
+Authentication dialog.
+
+For existing Aurora instances, the status of IAM authentication is displayed on
+the Configuration tab and can be enabled by modifying the database instance.
+
+Next, create the following IAM policy attached to a user whose credentials a
+Teleport process will be using to allow it to connect to the database:
+
+```json
+{
+   "Version": "2012-10-17",
+   "Statement": [
+      {
+         "Effect": "Allow",
+         "Action": [
+             "rds-db:connect"
+         ],
+         "Resource": [
+             "arn:aws:rds-db:<region>:<account-id>:dbuser:<db-cluster-resource-id>/*"
+         ]
+      }
+   ]
+}
+```
+
+Finally, connect to the database and create a database account with IAM auth
```

Comment:
I would add a link to https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/USER_ConnectToPostgreSQLInstance.html for newbie Postgres users

Reply:
I found https://aws.amazon.com/blogs/database/using-iam-authentication-to-connect-with-pgadmin-amazon-aurora-postgresql-or-amazon-rds-for-postgresql/ content also helpful as I learnt more about the specifics of IAM and Aurora Postgres

Reply:
Added a link below.